specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```diff
@@ -7215,6 +7310,51 @@
       },
       "description": "A skill to split a string into chunks of text."
     },
+    "CustomEntityLookupSkill": {
+      "x-ms-discriminator-value": "#Microsoft.Skills.Text.CustomEntityLookupSkill",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SearchIndexerSkill"
+        }
+      ],
+      "properties": {
+        "defaultLanguageCode": {
+          "$ref": "#/definitions/CustomEntityLookupSkillLanguage",
+          "description": "A value indicating which language code to use. Default is en."
+        },
+        "entitiesDefinitionUri": {
+          "type": "string",
+          "description": "Path to a JSON or CSV file containing all the target text to match against."
+        },
+        "inlineEntitiesDefinition": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CustomEntity"
+          },
+          "description": "The inline CustomEntity definition."
+        },
+        "globalDefaultCaseSensitive": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "A global flag for CaseSentitive. If CaseSensitive is not set in CustomEntity, this value will be the default value."
```

Comment:
What is the default value?

Reply:
The default value is null if customers don't provide one.